plugins/woocommerce/src/Internal/Admin/Suggestions/PaymentExtensionSuggestions.php

```diff
@@ -2867,6 +2869,39 @@ private function get_all_extensions_base_details(): array {
 					),
 				),
 			),
+			self::MONEI             => array(
+				'_type'       => self::TYPE_PSP,
+				'title'       => esc_html__( 'MONEI', 'woocommerce' ),
+				'description' => esc_html__( 'Accept Cards, Apple Pay, Google Pay, Bizum, PayPal, and many more payment methods in your store.', 'woocommerce' ),
+				'image'       => plugins_url( 'assets/images/onboarding/mollie.svg', WC_PLUGIN_FILE ),
```

Comment:
```suggestion
				'image'       => plugins_url( 'assets/images/onboarding/monei.svg', WC_PLUGIN_FILE ),
```
Not sure if it's the correct url, but just making a suggestion

Reply:
changed in [7002365](https://github.com/woocommerce/woocommerce/pull/55363/commits/7002365faeae7d9f976575d44f2399976ec2f621) . thanks